e2e/rdctl.e2e.spec.ts

```diff
@@ -731,7 +739,19 @@ test.describe('Command server', () => {
           stdout: '',
         });
         expect(stderr).toContain('Usage:');
-        expect(stderr.split(/\n/).filter(line => /^\s+--/.test(line)).length).toBe(34 - unsupportedOptions.length);
+        const options = stderr.split(/\n/)
+          .filter(line => /^\s+--/.test(line))
+          .map(line => (/\s+--([-.\w]+)\s/.exec(line) || [])[1])
+          .filter(line => line);
```

Comment:
(Writing this down to help me understand this better)
This should be equivalent to:
```suggestion
        const options = stderr.split(/\n/)
          .map(line => (/^\s+--([-.\w]+)\s/.exec(line) || [])[1])
          .filter(option => option);
```
In particular, for the last filter, the input is no longer a whole line.

Reply:
I'd rather leave this as is for more clarity that we're only interested in lines that start with <SPACE><DASH><DASH>